internal/handlers/pg/msg_delete.go

```diff
@@ -142,3 +129,25 @@ func (h *Handler) MsgDelete(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 
 	return &reply, nil
 }
+
+func (h *Handler) delete(ctx context.Context, resDocs []*types.Document, db string, collection string) (pgconn.CommandTag, error) {
+	var p pgdb.Placeholder
+	placeholders := make([]string, len(resDocs))
+	ids := make([]any, len(resDocs))
+	for i, doc := range resDocs {
+		placeholders[i] = p.Next()
+		id := must.NotFail(doc.Get("_id"))
+		ids[i] = must.NotFail(fjson.Marshal(id))
+	}
+
+	sql := fmt.Sprintf(
+		"DELETE FROM %s WHERE _jsonb->'_id' IN (%s)",
+		pgx.Identifier{db, collection}.Sanitize(), strings.Join(placeholders, ", "),
+	)
+	tag, err := h.pgPool.Exec(ctx, sql, ids...)
+	if err != nil {
+		// TODO check error code
```

Comment:
can it be done in this PR?

Reply:
It's doesn't looks like something related to this PR.